python/GangaDirac/test/Unit/Utilities/TestDiracUtilities.py

```diff
@@ -12,6 +9,7 @@
 def config_files():
     """
     Load the config files in a way similar to a full Ganga session
+    NB: This is taken to be only within the context of __this__ file!
```

Comment:
It's not obvious what the scope of the loaded config was when debugging the other tests in this file. The config is (I believe) loaded here and remains valid while the tests in this file are executed.